client/components/events/EventViewToggle/EventViewToggle.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import { EventCalendarView } from "../EventCalendarView"
+import { EventListView } from "../EventListView/EventListView"
+import { Event } from '../../../context'
+import { colours, device, fontWeight, desktopFontSize, mobileFontSize } from '../../../styles'
+import styled from 'styled-components'
+
+const LISTVIEW_ACTIVE = 0;
+const CALENDARVIEW_ACTIVE = 1;
```

Comment:
Grouping these two pieces of data into one would improve semantics.

`enum View {
    List,
    Calendar
}`

Reply:
```suggestion
const enum View { List, Calendar }
```

@jeffzh4ng GitHub code suggestions are super-convenient!

Reply:
😬